tests/lib/rules/jsx-key.js

```diff
@@ -47,46 +46,117 @@ ruleTester.run('jsx-key', rule, {
     {code: 'var App = () => <div />;'},
     {code: '[1, 2, 3].map(function(x) { return; });'},
     {code: 'foo(() => <div />);'},
-    {code: 'foo(() => <></>);', parser: parsers.BABEL_ESLINT},
-    {code: '<></>;', parser: parsers.BABEL_ESLINT}
```

Comment:
to ensure that this isn't a breaking change, please don't alter any existing test cases.

Reply:
I understand your argument but I would like to make a case for the change. 

I think moving tests like this is low risk, as you can see they are still listed in a `tests` object and they are being passed to the test runners. Also, the majority of tests are now being run with the 4 test runners improving the test suite.

If you prefer I can revert the change but I think the benefits outweigh the risk. What do you think?

Reply:
Fair enough.